main.tf

```
@@ -31,15 +39,55 @@ data "aws_sns_topic" "main" {
 # CloudWatch Event
 #
 
-resource "aws_cloudwatch_event_rule" "main" {
+resource "aws_cloudwatch_event_rule" "main-gov" {
```

The convention we usually try to stick to is that Terraform resources use underscores, AWS resources use hyphens, so this should be:
```suggestion
resource "aws_cloudwatch_event_rule" "main_gov" {
```

You'd need to change other references (and fix this for commercial too).